addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -478,6 +479,8 @@ registry.slider = publicWidget.Widget.extend({
             this.options.wysiwyg.odooEditor.observerUnactive("disable_controls");
             this.controlEls.forEach(controlEl => controlEl.removeAttribute("data-bs-slide"));
             indicatorEls.forEach(indicatorEl => indicatorEl.removeAttribute("data-bs-slide-to"));
+            this.el.setAttribute("data-bs-ride", "false");
+            Carousel.getOrCreateInstance(this.el)._config.ride = false;
```

Comment:
Hello @aans-odoo, 
I believe we should not modify the `_config` property. `_config` is a private property of the Bootstrap carousel object, and I don't think it's good practice to manually change private properties. 🙂 We should find another way to do it.